src/Symfony/Component/Form/Extension/Validator/Type/FormTypeValidatorExtension.php

```diff
@@ -58,9 +60,18 @@ public function configureOptions(OptionsResolver $resolver)
             'constraints' => array(),
             'invalid_message' => 'This value is not valid.',
```

Comment:
This must be change for BC:
```php
'invalid_message' => function (Options $options, ?string $previousValue) {
    return $options['legacy_error_messages'] ? 'This value is not valid.' : $previousValue;
},
```
We need a test case for this. Could you please add a test case in the BaseTypeTest to check the default value?